Otherwise version in .pc file has suffix -unknown which cbuild fails on.
diff --git a/autogen.sh b/autogen.sh
index 9b36158..d5cccd2 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -270,8 +270,8 @@
       rvd=$((0x$(echo ${rev} | dd bs=1 count=4 2>/dev/null)))
     else
       ingit=no
-      beta=yes
-      tmp="-unknown"
+      beta=no
+      tmp=""
       cid="0000000"
       rev="0000000"
       rvd="0"
